spago.dhall

```diff
@@ -17,11 +16,10 @@ You can edit this file as you like.
   , "pairs"
   , "prelude"
   , "psci-support"
```

Comment:
Should I put this into `test.dhall`? It's not exactly a test dependency though.

Reply:
Hm, let's leave it for now.